semana19/cookenu-projeto/src/services/HashManager.ts

```diff
@@ -0,0 +1,27 @@
+import * as bcrypt from "bcryptjs";
+
+
+export class HashManager{
+    async hash(s: string){
+        //para esconder o texto precisamos:
+        //do plaintext (que é o parâmetro testToHide);
+        //do nosso cost (que é o quanto vamos demorar pra esconder. Quanto maior mais escondido)
+        //do salt (que vai ser gerado aqui na função, a partir do cost ou rouds)
+        const cost = 12; //esse valor pode vir do .env
+        const salt = await bcrypt.genSalt(cost);
+        const result = await bcrypt.hash(s, salt);
```

Comment:
**dica:** use o .env  BCRYPT_COST = 12. Assim quem for testar a sua aplicação encontra todas as variáveis de ambientes necessárias para controlar a aplicação.